modules/lfs/http_client_test.go

```diff
@@ -211,36 +211,31 @@ func TestHTTPClientDownload(t *testing.T) {
 			expectederror: "TransferAdapter not found: ",
 		},
 		// case 5
-		{
-			endpoint:      "https://error-in-response-objects.io",
-			expectederror: "Object not found",
```

Comment:
this test wasn't as useful as it used to be since https://github.com/go-gitea/gitea/commit/09b56fc0690317891829906d45c1d645794c63d5 , the only place we used the lfs/http_client would ignore an error of type 404, so we were testing here that the client could still produce such an error but would always throw it away.